<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewPage</class>
 <widget class="QWidget" name="OverviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vl" stretch="1,1">
   <item>
    <layout class="QVBoxLayout" name="vl0">
	    /*  WalletOverview - send - disable */
     <item>
      <widget class="QFrame" name="control">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>

         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="WalletOverview">
            <property name="text">
             <string>Wallet Overview</string>
            </property>
           </widget>
          </item>

          <item>
           <spacer name="hs_send">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>

	  <item>
           <widget class="QPushButton" name="sendbutton">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
	  <item>
           <widget class="QPushButton" name="stakingbutton">
            <property name="text">
             <string>Disable Staking</string>
            </property>
           </widget>
          </item>

         </layout>
        </item>
       </layout>
      </widget>
     </item>
   /*  WalletOverview - send - disable END */

	    /*  START balance - staked - pending */
     <item>
      <widget class="QFrame" name="status_frame">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="statusHB">
          <item>
           <widget class="QFrame" name="available">
         <layout class="QGridLayout" name="statusGL">
          <item row="0" column="0" colspan="99">
                <widget class="QLabel" name="labelTextBalance">
                 <property name="text">
                  <string>Available</string>
                 </property>
                </widget>
	  </item>
          <item row="1" column="0">
                <widget class="QLabel" name="labelIntBalance">
                 <property name="alignment">
                  <set>Qt::AlignLeft|Qt::AlignBottom</set>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>Your current spendable balance</string>
                 </property>
                 <property name="text">
                  <string notr="true">0 I/O</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
	</item>
	  <item row="1" column="1">
                <widget class="QLabel" name="labelFracBalance">
                 <property name="alignment">
                  <set>Qt::AlignLeft|Qt::AlignBottom</set>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>Your current spendable balance</string>
                 </property>
                 <property name="text">
                  <string notr="true">0 I/O</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
	</item>
	  <item row="1" column="2">
                <widget class="QLabel" name="labelBalanceUnit">
                 <property name="text">
                  <string>IOC</string>
                 </property>
                </widget>
	  </item>
	  <item row="0" column="99" rowspan="2">
                <widget class="QLabel" name="balanceIcon">
		</widget>
	  </item>
</layout >
           </widget>
          </item>

	  <item>
           <widget class="QFrame" name="pending">
         <layout class="QGridLayout" name="unconfirmedGL">
          <item row="0" column="0" colspan="99">
                <widget class="QLabel" name="labelTextUnconfirmed">
                 <property name="text">
                  <string>Pending</string>
                 </property>
                </widget>
	  </item>
          <item row="1" column="0">
                <widget class="QLabel" name="labelIntUnconfirmed">
                 <property name="alignment">
                  <set>Qt::AlignLeft|Qt::AlignBottom</set>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>Total of transactions that have yet to be confirmed, and do not yet count toward the current balance</string>
                 </property>
                 <property name="text">
                  <string notr="true">0 I/O</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
	</item>
	  <item row="1" column="1">
                <widget class="QLabel" name="labelFracUnconfirmed">
                 <property name="alignment">
                  <set>Qt::AlignLeft|Qt::AlignBottom</set>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>Total of transactions that have yet to be confirmed, and do not yet count toward the current balance</string>
                 </property>
                 <property name="text">
                  <string notr="true">0 I/O</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
	</item>
	  <item row="1" column="2">
                <widget class="QLabel" name="labelUnconfirmedUnit">
                 <property name="text">
                  <string>IOC</string>
                 </property>
                </widget>
	  </item>
	  <item row="0" column="99" rowspan="2">
                <widget class="QLabel" name="pendingIcon">
		</widget>
	  </item>
            </layout>
	   </widget>
          </item>
	  <item>
           <widget class="QFrame" name="staked">
         <layout class="QGridLayout" name="stakedGL">
          <item row="0" column="0" colspan="99">
                <widget class="QLabel" name="labelTextStaked">
                 <property name="text">
                  <string>Staked</string>
                 </property>
                </widget>
	  </item>
          <item row="1" column="0">
                <widget class="QLabel" name="labelIntStaked">
                 <property name="alignment">
                  <set>Qt::AlignLeft|Qt::AlignBottom</set>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>Total of coins that was staked, and do not yet count toward the current balance</string>
                 </property>
                 <property name="text">
                  <string notr="true">0 I/O</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
	</item>
	  <item row="1" column="1">
                <widget class="QLabel" name="labelFracStaked">
                 <property name="alignment">
                  <set>Qt::AlignLeft|Qt::AlignBottom</set>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>Total of coins that was staked, and do not yet count toward the current balance</string>
                 </property>
                 <property name="text">
                  <string notr="true">0 I/O</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
	</item>
	  <item row="1" column="2">
                <widget class="QLabel" name="labelStakedUnit">
                 <property name="text">
                  <string>IOC</string>
                 </property>
                </widget>
	  </item>
	  <item row="0" column="99" rowspan="2">
                <widget class="QLabel" name="stakedIcon">
		</widget>
	  </item>
            </layout>
           </widget>
          </item>

         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
	    /*  END balance - staked - pending */
	    /*  Recent Transactions - send - disable */
   <item>
    <layout class="QVBoxLayout" name="vl0">
     <item>
      <widget class="QFrame" name="recenttransactions">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="textRecentTrans">
          <property name="text">
           <string>Recent Transactions</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   /*  recenttransactions - send - disable END */


   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="walletbalance">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="txviewlayout">
       <item>
        <widget class="QWidget" name="txv">
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
